securedrop/models.py

```diff
@@ -327,10 +327,12 @@ def set_hotp_secret(self, otp_secret):
 
     @property
     def totp(self):
+        # type: () -> str
```

Comment:
These type annotations are incorrect. `TOTP` and `HOTP` return objects that have to be called to create or verify tokens.

Reply:
Ah, these went in by mistake :(